install-ubuntu.sh

```diff
@@ -21,13 +21,13 @@
       && apt-get install -y apt-transport-https)
 
   # add heroku repository to apt
-  echo "deb https://cli-assets.heroku.com/apt ./" > /etc/apt/sources.list.d/heroku.list
+  echo "deb [signed-by=/usr/share/keyrings/heroku-archive-keyring.gpg] https://cli-assets.heroku.com/apt ./" > /etc/apt/sources.list.d/heroku.list
 
   # remove toolbelt
   (dpkg -s heroku-toolbelt 1>/dev/null 2>/dev/null && (apt-get remove -y heroku-toolbelt heroku || true)) || true
 
   # install heroku's release key for package verification
-  curl https://cli-assets.heroku.com/apt/release.key | apt-key add -
+  curl https://cli-assets.heroku.com/apt/release.key | sudo gpg --dearmor -o /usr/share/keyrings/heroku-archive-keyring.gpg
```

Comment:
I don't think you need `sudo` here as superuser access has been set up earlier in the script.

Reply:
Oh indeed, I've copied the usual command without thinking about that, thank you!